packages/colony-contract-loader-network/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@colony/colony-contract-loader-network",
-  "version": "1.0.0",
+  "version": "1.0.1",
```

Comment:
The version is incremented here with the expectation that we'll publish it afterwards; if I could write straight to `develop`, I wouldn't add this here.